MachObfuscatorTests/MachTests/Mach+Replacing_propertyattributes_Tests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Mach+Replacing_propertyattributes_Tests.swift
+//  MachObfuscatorTests
+//
+
+import Foundation
+import XCTest
+
+private extension ObjcClass {
+    func property(_ name: String) -> ObjcProperty? {
+        return properties.first { $0.name.value == name }
+    }
+}
+
+class Mach_Replacing_propertyattributes_Tests: XCTestCase {
+    func test_shouldReplaceClassnamesInMach_WhenMatchingMangling() {
+        // Given
+        var sut = try! Image.load(url: URL.machoIos12_0Executable)
+
+        // Prepare test obfuscation configuration satisfying requirements of replaceSymbols
+        let emptyTrie = Trie(exportsSymbol: false, labelRange: 0 ..< 0, label: [], children: [])
+        let firstImportEntry = sut.machs[0].importStack[0]
+        let symbolDylib = sut.machs[0].dylibs[firstImportEntry.dylibOrdinal - 1]
+
+        // In normal case NSString would not be obfuscated. In test we use it because it is referenced in property types
+        let map = SymbolManglingMap(selectors: [:], classNames: ["NSString": "ObfStrng"], exportTrieObfuscationMap: [
+            URL.machoIos12_0Executable: [
+                sut.machs[0].cpu.asCpuId:
+                    (unobfuscated: emptyTrie,
+                     obfuscated: emptyTrie),
+            ],
+        ])
+        var paths = ObfuscationPaths()
+        paths.resolvedDylibMapPerImageURL = [URL.machoIos12_0Executable: [symbolDylib: URL(fileURLWithPath: "/tmp/testlib")]]
```

Comment:
The test looks ok. I am only not sure if this line is really needed. We should probably not test export-trie obfuscation here.

Reply:
As it turns out it is not needed - resolved dylibs may be empty map, but `paths.resolvedDylibMapPerImageURL` must contain entry for image being obfuscated.
I simplified this test and also others that used similar construct. I removed also duplications of `emptyTrie`.

Reply:
Thanks!